src/main/java/dojo/portfolio/stringcalculator/illegalArgumentException.java

```diff
@@ -0,0 +1,6 @@
+package dojo.portfolio.stringcalculator;
+
+public class illegalArgumentException extends Throwable {
```

Comment:
classes start with capital letter by convention

Reply:
Thanks, I just corrected it.